magnetis/astro-native
# bugfix(icons): fix icons svg paths [CONFIA-859]

# What


Fix Icons svg paths.
# Why
To be rendered correctly.


# How
Changing svg paths and props of `ArrowChevronDown`, `FingerprintCheck`, `FingerprintError` and `FingerprintPadlock` icons.


# Sample


![Screenshot 2023-03-08 at 17 24 46](https://user-images.githubusercontent.com/35939685/223841576-cb805646-da73-44bb-b3c6-29418cd66cd6.png)

![Screenshot 2023-03-08 at 17 24 04](https://user-images.githubusercontent.com/35939685/223841431-e7ed2bfd-e2c9-4f4f-86d1-19707537c99b.png)
![Screenshot 2023-03-08 at 17 24 15](https://user-images.githubusercontent.com/35939685/223841464-659f6eed-b99c-47fa-8d0d-bad237fadcea.png)
![Screenshot 2023-03-08 at 17 24 26](https://user-images.githubusercontent.com/35939685/223841514-6eb90cfd-ac09-4424-be33-04bdf45c91a6.png)

# QA

1. Start the storybook;
2. Run your preferred emulator;
3. In the menu, choose Icons and select `ArrowChevronDown`, `FingerprintCheck`, `FingerprintError` and `FingerprintPadlock` icons and see if they are rendered properly.








[CONFIA-859](https://produtomagnetis.atlassian.net/browse/CONFIA-859)


[CONFIA-859]: https://produtomagnetis.atlassian.net/browse/CONFIA-859?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
